### PR TITLE
assignment 1

### PR DESCRIPTION
**Pull request by lia7lx**

### CODE REVIEW

File: assignment-1-sql/xiang-li.sql [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+		 WHEN EMPLOYMENT_LENGTH IS NULL OR EMPLOYMENT_LENGTH < 0 THEN 'N/A'
+		 WHEN EMPLOYMENT_LENGTH BETWEEN 0 AND 5 THEN '0-5'
+		 WHEN EMPLOYMENT_LENGTH BETWEEN 6 AND 11 THEN '6-11'
+		 WHEN EMPLOYMENT_LENGTH BETWEEN 12 AND 17 THEN '12-17'
+		 WHEN EMPLOYMENT_LENGTH BETWEEN 18 AND 23 THEN '18-23'
+		 WHEN EMPLOYMENT_LENGTH >= 24 THEN '24+' END AS EMPLOYMENT_LENGTH_RANGE
+	FROM RDS_DATA.DRIVERS 
+	WHERE ACTIVE = TRUE
+)
+SELECT SAR.EMPLOYMENT_LENGTH_RANGE, SUM(SM.TOTAL_DELIVERIES_COUNT)/SUM(SM.TOTAL_MPI) AS AVG_MPI
```
Comment (shaolinhoward) [inline assignment-1-sql/xiang-li.sql]: would be nice to put a `ROUND()` surrounding the average calculation

File: assignment-1-sql/xiang-li.sql [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+	JOIN RDS_DATA.ORDERS O ON OD.ORDER_ID = O.ID
+	WHERE OD.WAREHOUSE_ID = 1000
+)
+WHERE RN = 1
+ORDER BY CREATED_AT ASC
+LIMIT 10;
+
+
+SELECT DATEADD('year', 5, to_timestamp_ntz(convert_timezone('UTC', 'US/Pacific', CREATED_AT))::DATE)
+FROM ECLIPSE.USERS
```
Comment (shaolinhoward) [inline assignment-1-sql/xiang-li.sql:130]: don't think everyone has access to this table (I don't) but the query looks good

**Review by shaolinhoward — APPROVED** [review]
nice job!

**Comment by lia7lx** [comment]
@shaolinhoward  I rounded the avg_mpi. If it looks good to you, could you help me merge my branch? I am no longer authorized to merge.

**Pull request merged**